src/services/ui/src/pages/dashboard/index.tsx

```diff
@@ -11,11 +11,14 @@ import {
   FilterChips,
   FilterDrawerProvider,
 } from "@/components/Opensearch";
-import { Button } from "@/components/Inputs";
 import { ROUTES } from "@/routes";
+import { Button } from "@/components/Inputs";
 import { useUserContext } from "@/components/Context/userContext";
 import { useMemo } from "react";
 
+const buttonStyles =
+  "text-white bg-primary inline-block border-none px-10 py-2 rounded cursor-pointer";
```

Comment:
This string doesn't have to be a variable. We utilize in-line strings via the `className` value so our styles remain close to their consumption/use. Saves us from chasing down variables later on.